definitions/classnames_v2.x.x/flow_>=v0.23.x/classnames_v2.x.x.js

```diff
@@ -1,5 +1,8 @@
+
+type Classes = string | [class: string]: bool } | Array<string>
```

Comment:
Again I thought I had to use
`declare var x = ...
declare type TypeName = ...`

In any case, I'll namespace this.